### ISSUE TITLE
Conversion Error

### ISSUE THREAD

**Issue by HOKORISAMA** [TheVNConnoisseur/croCKer#1]
It fails to reconvert the files from txt to srp.
[srp.zip](https://github.com/user-attachments/files/18218551/srp.zip)

**Comment by TheVNConnoisseur** [comment]
> It fails to reconvert to the files from txt to srp. [srp.zip](https://github.com/user-attachments/files/18218551/srp.zip)

Hi @HOKORISAMA, could you please tell me what game is it for, and more precisely, if you made any changes to the script? If so, let me know where so I can check it in detail.

**Comment by HOKORISAMA** [comment]
> > It fails to reconvert to the files from txt to srp. [srp.zip](https://github.com/user-attachments/files/18218551/srp.zip)
> 
> Hi @HOKORISAMA, could you please tell me what game is it for, and more precisely, if you made any changes to the script? If so, let me know where so I can check it in detail.

It's [The shemale](https://vndb.org/v26548).
And no I didn't made any changes.

**Comment by TheVNConnoisseur** [comment]
Ok, so by taking a quick glance onto the files, the problem comes from the fact that the conflicting files don't even get an extension when even using PACkager, so we are dealing with a newer version of SRP files or straight out another kind of file. For the mean time, don't even bother modifying those specific files, because they need to be investigated.

And yes, you can still modify the identified files because those are still standard SRP and VAR files (they'll recompile fine).

**Comment by TheVNConnoisseur** [comment]
> > > It fails to reconvert to the files from txt to srp. [srp.zip](https://github.com/user-attachments/files/18218551/srp.zip)
> > 
> > 
> > Hi @HOKORISAMA, could you please tell me what game is it for, and more precisely, if you made any changes to the script? If so, let me know where so I can check it in detail.
> 
> It's [The shemale](https://vndb.org/v26548). And no I didn't made any changes.

Ok, so by taking a quick glance onto the files, the problem comes from the fact that the conflicting files don't even get an extension when even using PACkager, so we are dealing with a newer version of SRP files or straight out another kind of file. For the mean time, don't even bother modifying those specific files, because they need to be investigated.

And yes, you can still modify the identified files because those are still standard SRP and VAR files (they'll recompile fine).

**Comment by HOKORISAMA** [comment]
> > > > It fails to reconvert to the files from txt to srp. [srp.zip](https://github.com/user-attachments/files/18218551/srp.zip)
> > > 
> > > 
> > > Hi @HOKORISAMA, could you please tell me what game is it for, and more precisely, if you made any changes to the script? If so, let me know where so I can check it in detail.
> > 
> > 
> > It's [The shemale](https://vndb.org/v26548). And no I didn't made any changes.
> 
> Ok, so by taking a quick glance onto the files, the problem comes from the fact that the conflicting files don't even get an extension when even using PACkager, so we are dealing with a newer version of SRP files or straight out another kind of file. For the mean time, don't even bother modifying those specific files, because they need to be investigated.
> 
> And yes, you can still modify the identified files because those are still standard SRP and VAR files (they'll recompile fine).

You mean I should leave the files that do not have any extension, right?

**Comment by TheVNConnoisseur** [comment]
Yes, you can select those files with this program no problem, since it will ignore them to begin with since they are extensionless.
Just edit the .txt generated files with what you need, and just compile them again.
In case that the files when compiled don't load correctly in-game, make sure to first test the game without making any edits to the script, in order to know if your edits are the culprit or the compilation process in itself is improperly done.

**Comment by TheVNConnoisseur** [comment]
Yes @HOKORISAMA, you can select those files with this program no problem, since it will ignore them to begin with since they are extensionless.
Just edit the .txt generated files with what you need, and just compile them again.
In case that the files when compiled don't load correctly in-game, make sure to first test the game without making any edits to the script, in order to know if your edits are the culprit or the compilation process in itself is improperly done.
